### ISSUE TITLE
SOURCE_DATE_EPOCH causes test failures

### ISSUE THREAD

**Issue by eli-schwartz** [asciidoc/asciidoc-py3#131]
Running the entire build/test/install cycle for asciidoc itself with this environment variable set, causes the testsuite to see surprising output. Many tests fail with output like this:

```
+++ tests/data/oldtables-html4.html
--- got
- 2020-07-28 00:58:43 UTC
+ 2002-11-25 00:37:42 UTC

+++ tests/data/oldtables-xhtml11.html
--- got
- 2020-07-28 00:58:43 UTC
+ 2002-11-25 00:37:42 UTC
```

Solution: tests/testasciidoc.py should account for this in mock_localtime. Most likely, you could remove this entirely, and rely on defining SOURCE_DATE_EPOCH in the test setup. Otherwise, it will need to be deleted from os.environ. The former would provide free testing for the feature though. :)

**Comment by MasterOdin** [comment]
To be clear, you've got `SOURCE_DATE_EPOCH` set beforehand, and then when you run the test suite and it causes test failures?

**Comment by eli-schwartz** [comment]
Yeah, sorry, I managed to lose that part of my sentence when filing this issue...

I ran into this while preparing the archlinux package update to 9.0.2 (yay python3) and I realized "hey, there is a testsuite here, it would be fantastic to run it while building the package".

This is not very urgent as it only prevents me from _adding_ the testsuite to my packaging recipe. But I've solved that anyway by running it with `env -u SOURCE_DATE_EPOCH make test`, so even that is not an immediate issue.
